### ISSUE TITLE
Upload error

### ISSUE THREAD

**Issue by YegorDan** [LibreTime/libretime#289]
I get an error when uploading a track: An error occured while processing your upload. Please try agan in a few minutes.

zendphp.log

```
2017-09-08T20:25:28+03:00 INFO (6): airtime.dropthebass.ru Moving file /tmp/phpCVe1cw to /srv/airtime/stor/organize/A Sides Makoto - Breathe.mp3
2017-09-08T20:25:28+03:00 INFO (6): Methods with the same name as their class will not be constructors in a future version of PHP; ProxyStorageBackend has a deprecated constructor
2017-09-08T20:25:28+03:00 ERR (3): airtime.dropthebass.ru [MediaController.php:160 - postAction()] - Methods with the same name as their class will not be constructors in a future version of PHP; ProxyStorageBackend has a deprecated constructor
#0 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Log.php(636): exception_error_handler(8192, 'Methods with th...', '/usr/share/airt...', 9, Array)
#1 /usr/share/airtime/php/vendor/composer/ClassLoader.php(444): Zend_Log->errorHandler(8192, 'Methods with th...', '/usr/share/airt...', 9, Array)
#2 /usr/share/airtime/php/vendor/composer/ClassLoader.php(444): include()
#3 /usr/share/airtime/php/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile('/usr/share/airt...')
#4 [internal function]: Composer\Autoload\ClassLoader->loadClass('ProxyStorageBac...')
#5 /usr/share/airtime/php/airtime_mvc/application/services/MediaService.php(41): spl_autoload_call('ProxyStorageBac...')
#6 /usr/share/airtime/php/airtime_mvc/application/models/airtime/CcFiles.php(160): Application_Service_MediaService::importFileToLibrary('https://airtime...', '/tmp/phpCVe1cw', 'A Sides Makoto ...', 1, false)
#7 /usr/share/airtime/php/airtime_mvc/application/models/airtime/CcFiles.php(99): CcFiles::createAndImport(Array, '/tmp/phpCVe1cw', 'A Sides Makoto ...')
#8 /usr/share/airtime/php/airtime_mvc/application/modules/rest/controllers/MediaController.php(144): CcFiles::createFromUpload(Array)
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): Rest_MediaController->postAction()
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('postAction')
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#13 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#14 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#15 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#16 {main}
2017-09-08T20:25:34+03:00 INFO (6): airtime.dropthebass.ru Checking autoplaylist poll
```

PHP 7.0
Ubuntu 16.04

How i can fix it?

**Comment by hairmare** [comment]
I just pushed a fix in #290, you probably also can work around this error by setting a lower error level in php (ie. one that does not containing the info level).

**Comment by YegorDan** [comment]
Thanks alot! It works now!
